unite_contranges.py

```diff
@@ -0,0 +1,146 @@
+#Allison Towner
+#09 October 2020
+
+#continuum_selections.py in analysis code is located at:
+#in ALMA_IMF/reduction/analysis/
+
+def merge_contdotdat(field='',band=''):
```

Comment:
```suggestion
def merge_contdotdat(field,band):
```
if the defaults aren't valid, don't specify defaults